src/main/java/seedu/address/model/student/Student.java

```diff
@@ -61,7 +61,9 @@ public boolean isSameStudent(Student otherStudent) {
 
         return otherStudent != null
                 && otherStudent.getName().equals(getName())
-                && otherStudent.getPhone().equals(getPhone());
+                && (otherStudent.getPhone().equals(getPhone())
+                    && otherStudent.getSchool().equals(getSchool())
+                        && otherStudent.getYear().equals(getYear()));
```

Comment:
Just a minor nitpick for neatness, but I think these 2 lines could be on the same level of indentation as line 64.

Reply:
okay done thanks